ui/lib/apps/SlowQuery/utils/tableColumns.tsx

```diff
@@ -28,14 +32,20 @@ function commonColumnName(fieldName: string): any {
   return <TextWithInfo.TransKey transKey={`slow_query.fields.${fieldName}`} />
 }
 
+//////////////////////////////////////////
+// Notice:
+// The key field value in the following methods is case-sensitive
+// They should keep the same as the column name in the slow query table
+// Ref: pkg/apiserver/slowquery/queries.go SlowQuery struct
+
 function sqlColumn(
   _rows?: { query?: string }[], // used for type check only
   showFullSQL?: boolean
 ): IColumn {
   return {
     name: commonColumnName('sql'),
-    key: 'sql',
-    fieldName: 'sql',
+    key: 'Query',
+    fieldName: 'query', // fieldName is used for sort
```

Comment:
Remove the comment. The statement use fieldName to sort in local while slow query doesn't. It sorts in the backend.